src/main/java/ca/uhn/fhir/jpa/starter/common/FhirServerConfigCommon.java

```diff
@@ -253,6 +253,10 @@ public JpaStorageSettings jpaStorageSettings(AppProperties appProperties) {
 		jpaStorageSettings.setBundleBatchPoolSize(appProperties.getBundle_batch_pool_size());
 		jpaStorageSettings.setBundleBatchPoolSize(appProperties.getBundle_batch_pool_max_size());
 
+		// Set store meta source information
+		ourLog.info("Server configured to Store Meta Source:" + appProperties.getStore_meta_source_information());
```

Comment:
This should probably be a debug level logger to keep the startup noise to a dull roar.

```suggestion
		ourLog.debug("Server configured to Store Meta Source: {}", appProperties.getStore_meta_source_information());
```